po/de.po

```diff
@@ -1,20 +1,21 @@
 #
 # Cvfosammmm <r.griesel@gmail.com>, 2020.
+# Stephan Lachnit <stephanlachnit@protonmail.com>, 2020.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: setzer\n"
-"Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2020-04-28 22:57+0200\n"
-"PO-Revision-Date: 2020-04-28 23:57+0200\n"
-"Last-Translator: Cvfosammmm <r.griesel@gmail.com>\n"
-"Language-Team: German <r.griesel@gmail.com>\n"
```

Comment:
I suggest to removing the email in this row, since it doesn't really help anyone